i18n/config.py

```diff
@@ -27,6 +27,7 @@ class Configuration(object):
         'source_locale': 'en',
         'third_party': [],
         'TRANSIFEX_URL': 'https://www.transifex.com/open-edx/edx-platform/',
+        'edx_lang_map': {}
```

Comment:
Use a trailing comma to make diffs nicer when the next line is eventually added.